src/CoreBundle/Resources/config/cronjobs.xml

```diff
@@ -16,7 +16,6 @@
         </service>
 
         <service class="TCMSCronJob_ClearCompleteCache" id="chameleon_system_core.cronjob.clear_complete_cache_cronjob" shared="false">
-            <argument type="service" id="chameleon_system_cms_cache.cache"/>
```

Comment:
Isn't there / shouldn't there be  a different issue/PR for this?

Reply:
I couldn't call any SanityChecks (seems Symfony 3 is more strict here), so I fixed it directly. Shouldn't affect a separate bugfix PR.

Reply:
I mean mustn't this be fixed before in 6.2 (and master)?

Reply:
Yes, the PR you proposed will fix the problem in 6.2.x and we will merge the fix into master. With this separate fix (which is the same fix and cost close to no time) master works before the merge and the merge should recognize it's the same change and not raise a conflict. So it's better to have it in I think.